dynatrace-operator-google-marketplace/chart/dynatrace-operator/values.yaml

```diff
@@ -30,7 +30,7 @@ activeGate:
 kubernetesMonitoring:
   enabled: true
   replicas: 1
-  serviceAccountName: "dynatrace-kubernetes-monitoring"
+  serviceAccountName: ""
```

Comment:
This is unused then?

Reply:
Yes, I think we wouldn't need it since in the schema.yaml we define the ServiceAccounts which then automatically get created and can be used by .Values.NameProvidedAtTheEndOfTheFile

Reply:
Will leave it to you what to do with it.

Reply:
Will keep it for now and get rid of it in the next release